qt/widgets/common/src/WorkspacePresenter/WorkspaceTreeWidgetSimple.cpp

```diff
@@ -101,13 +104,15 @@ void WorkspaceTreeWidgetSimple::popupContextMenu() {
       menu->addMenu(plotSubMenu);
       menu->addSeparator();
       menu->addAction(m_showData);
+      menu->addAction(m_showAlgorithmHistory);
```

Comment:
I know you didn't add this option but while we're messing with the menu could we move the `Sample Logs` item from where it currently is to underneath "Show Instrument" and re-label it "Show Sample Logs"? It also only applies to `MatrixWorkspace` types so it doesn't need to be added to the items for a table.

Reply:
@martyngigg :unamused: #25092

Reply:
:man_facepalming: My bad. Sorry!